src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuPaymentRequestServiceImpl.java

```diff
@@ -415,4 +418,65 @@ public String getPaymentRequestNoteTargetObjectId(String documentNumber) {
         return ((CuPaymentRequestDao) paymentRequestDao).getObjectIdByPaymentRequestDocumentNumber(documentNumber);
     }
 
+    /**
+     * This implementation delegates to the is-quantity or is-within-amount-limit methods
+     * to make the determination.
+     * 
+     * This logic separation into multiple methods was largely done to help with unit testing.
+     * 
+     * @see edu.cornell.kfs.module.purap.document.service.CuPaymentRequestService#purchaseOrderForPaymentRequestIsNonQuantityOrWithinAutoApproveAmountLimit(
+     * org.kuali.kfs.module.purap.document.PaymentRequestDocument)
+     */
+    @Override
+    public boolean purchaseOrderForPaymentRequestIsNonQuantityOrWithinAutoApproveAmountLimit(PaymentRequestDocument document) {
+        return !purchaseOrderForPaymentRequestHasQuantityItems(document) || purchaseOrderForPaymentRequestIsWithinAutoApproveAmountLimit(document);
+    }
+
+    /**
+     * This implementation uses the "DEFAULT_POS_APRVL_LMT" parameter to get the cutoff value
+     * for which the PO amount can allow for automatic PREQ approval.
+     * 
+     * @see edu.cornell.kfs.module.purap.document.service.CuPaymentRequestService#isPurchaseOrderWithinAmountLimitForPaymentRequestAutoApprove(
+     * org.kuali.kfs.module.purap.document.PaymentRequestDocument)
+     */
+    @Override
+    public boolean purchaseOrderForPaymentRequestIsWithinAutoApproveAmountLimit(PaymentRequestDocument document) {
+        PurchaseOrderDocument po = document.getPurchaseOrderDocument();
+        String amountParameterValue = parameterService.getParameterValueAsString(
+                PaymentRequestDocument.class, PurapParameterConstants.PURAP_DEFAULT_NEGATIVE_PAYMENT_REQUEST_APPROVAL_LIMIT);
+        KualiDecimal amountLimit = new KualiDecimal(amountParameterValue);
+        boolean withinLimit = po.getFinancialSystemDocumentHeader().getFinancialDocumentTotalAmount().isLessThan(amountLimit);
+        LOG.info("PayReq " + document.getDocumentNumber()
+                + (withinLimit
+                        ? " has the potential for auto-approval because the amount on the associated PO is below the limit of "
+                        : " cannot be auto-approved because the amount on the associated PO matches or exceeds the limit of ")
+                + amountLimit.toString());
+        
+        return withinLimit;
+    }
+
+    /**
+     * This implementation just checks the first item since CU does not allow a mix of
+     * quantity and non-quantity items on the same document.
+     * 
+     * @see edu.cornell.kfs.module.purap.document.service.CuPaymentRequestService#purchaseOrderForPaymentRequestHasQuantityItems(
+     * org.kuali.kfs.module.purap.document.PaymentRequestDocument)
+     */
+    @Override
+    public boolean purchaseOrderForPaymentRequestHasQuantityItems(PaymentRequestDocument document) {
+        return !((PurchaseOrderItem) document.getPurchaseOrderDocument().getItems().get(0)).isNoQtyItem();

```

Comment:
I know that REQS are required to have at least one item, so I believe the POs will also have an item. However, I'll work on adding a safeguard in there in case the list does end up being empty for some odd reason.